reference/special-variables/def.markdown

```diff
@@ -31,3 +31,14 @@ bundle common def
   # ...
 }
 ```
+
+### def.jq
+
+This variable gives a convenient way to invoke
+[jq][https://stedolan.github.io/jq/] for the `mapdata()` function in `json_pipe`
+mode and elsewhere. Note the below is the **default** value that you can
```

Comment:
Is this default in C or in `bundle common def` of the Masterfiles Policy Framework we should probably specify.